docs/integration-tests.md

```diff
@@ -54,5 +54,5 @@ Then in the test you require the mock you can use the function name (e.g. `moto_
 
 Because we use aws profiles we require the profile name to be set up in boto3 for testing.  A "mocked" aws profile is set up in `integration_tests/conftest.py` in the `aws_credentials` fixture (this is why the other moto fixtures have the `aws_credentials` parameter).  If the code under test needs an aws profile name, please use "mocked".
 
-[back to Automated Tests](automated-tests.md)  
```

Comment:
the double spaces at the end of this line force a new line when the README is viewed in the browser - so they need to stay and be excluded from pre-commit

Reply:
excluded